src/prometheus/deploy/prometheus-configmap.yaml.template

```diff
@@ -36,6 +36,13 @@ data:
     scrape_configs:
     - job_name: 'node_exporter'
       scrape_interval: {{ prom_info['scrape_interval']|default(30) }}s
+      {% if clusterinfo['prometheusinfo']['use_tls'] -%}
+      tls_config:
+        ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
```

Comment:
This path is a volume or what?

Reply:
a volume mounted by k8s for service account auth

Reply:
Ok, then, I think it's inappropriate to config under `prometheus`.

Reply:
yes, watchdog's method is a little different with prometheus. it need reconfig http url access.

Reply:
make config as a global cluster wide config which prometheus watchdog webportal future use